### PR TITLE
Parameter Service In-Depth Jasmine Tests Fixes

### PR DESCRIPTION
**Pull request by mfuroyama**
This code review is pretty much just FYI, @creindle. Take a look, for your own piece of mind, then approve and I'll merge in.

The test failures pretty much came down to two issues:
1. There was actually a mismatch in the way labels were stored between the `Parameter` and the `ParameterDefinition` collections. in the `Parameters` collection, labels were stored as `ORK SYSTEM ENABLE_DISABLE` where they're stored as `ORK SYSTEM ENABLE/DISABLE` in the `ParameterDefinition` collection.
2. The location-type test cases for `ORWPCE ASK ENCOUNTER UPDATE` were slightly off. Changing the test cases to match the existing data records fixed the failures.

### CODE REVIEW

File: services/parameterService.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ class ParameterService extends BaseService {
             }
 
             // Make sure we're only pulling parameters for the station specified in the context
-            const query = { label: name };
+            const query = { label: name.split('_').join('/') };
```
Comment (creindle) [inline services/parameterService.js]: I thought you fixed this already?? :O Ok, I should've known!

Comment (creindle) [reply services/parameterService.js]: I distinctly remember you talking about how you fixed it thats why.

File: services/tests/parameterServiceInDepthALL-spec.js [hunk 2]
```diff
@@ -43,6 +43,15 @@ describe('Parameter Services Jasmine "ALL" tests', () => {
         stationId: '9999',
         locationId: 'urn:vista:9999:44:123',
     };
+    const contextICUS = {
```
Comment (creindle) [inline services/tests/parameterServiceInDepthALL-spec.js]: What is ICUS?

File: services/tests/parameterServiceInDepthSingle-spec.js [hunk 3]
```diff
@@ -1240,14 +1248,14 @@ describe('Parameter Services Jasmine "Single Entity" tests', () => {
     * Entity Length 1
     * Value (type)  '5, 6, 7' (set of codes (0:Primry/Data Needed; 1:Primry/Outpt; 2:Primry Always; 3:Data Needed; 4:Outpt; 5:Always; 6:Never; 7:Disable))
     */
-    xit('Expect ParameterService to retrieve "ORWPCE ASK ENCOUNTER UPDATE" correctly for PACKAGE Entity Type', (done) => {
-        ParameterService.withContext(context).get('ORWPCE ASK ENCOUNTER UPDATE', { entity: 'LOC' })
+    it('Expect ParameterService to retrieve "ORWPCE ASK ENCOUNTER UPDATE" correctly for PACKAGE Entity Type', (done) => {
+        ParameterService.withContext(contextICUS).get('ORWPCE ASK ENCOUNTER UPDATE', { entity: 'LOC' })
```
Comment (creindle) [inline services/tests/parameterServiceInDepthSingle-spec.js]: Ooooh, should I have looked closer into the MongoDB for a correct user.id? Or is this something new?

File: services/parameterService.js (hunk 1 above)
Comment (mfuroyama) [reply services/parameterService.js]: Hmm, yeah, there's a couple places where the comparison is made.

Eventually, we might actually go through and normalize the data sets, but bottom line is that it's good we caught this here!  This is exactly the reason why we wanted to build these tests!

File: services/tests/parameterServiceInDepthALL-spec.js (hunk 2 above)
Comment (mfuroyama) [reply services/tests/parameterServiceInDepthALL-spec.js]: `ICUS` is short for `ICU-S`, which is the label value of the location (44) entity that contains a valid value. The original `context` object location entity doesn't contain values for the named parameter

File: services/tests/parameterServiceInDepthSingle-spec.js (hunk 3 above)
Comment (mfuroyama) [reply services/tests/parameterServiceInDepthSingle-spec.js]: See the above comment. I created a new session context object for this particular test to get the test specs to return values.

File: services/parameterService.js (hunk 1 above)
Comment (creindle) [reply services/parameterService.js]: Awesome :)

**Pull request merged**